README.md

```diff
@@ -1,44 +1,48 @@
 <a name="readme-top"></a>
 
 [![MIT License][license-shield]][license-url]
-[![Download in the Chrome Web Store][chrome-shield]][chrome-url]
-[![Download in the Firefox Add-on Store][firefox-shield]][firefox-url]
 
 <!-- PROJECT LOGO -->
-<br />
+
 <div align="center">
-  <a href="https://github.com/AndrewWalsh/openapi-devtools">
-    <img src="resources/logo.svg" alt="Open API dev tools" width="300" height="250">
+  <a href="https://github.com/Superflows-AI/openapi-llm">
+    <img src="resources/logo.svg" alt="LLM for OpenAPI DevTool" width="300" height="250">
   </a>
+</div>
 
+<br />
 
-  <p align="center" style="max-width: 600px;">
-    Effortlessly discover API behaviour with a browser extension that automatically generates OpenAPI specifications in real time for any app or website.
-    <br />
-    <br />
-    <a href="https://awalsh.io/posts/openapi-devtools/">Read More</a>
-    ·
-    <a href="https://github.com/AndrewWalsh/openapi-devtools/issues">Report Bug</a>
-    ·
-    <a href="https://forms.gle/fA2EtvamUrsRxXKYA">Give Feedback</a>
-  </p>
-  </p>
-</div>
+ Generate a OpenAPI Specification in real time for any app or website, including descriptions of endpoints and parameters.
+
+<br />
+
+A browser extension that discovers API behaviour via your interactions with a product or website, and then uses an LLM to describe the API endpoints discovered.
+
+<br />
+
+This project is a fork of OpenAPI Devtools. Check out the <a href="https://github.com/AndrewWalsh/openapi-devtools"> original tool here. </a>
```

Comment:
Personally would only include this (if at all) lower down since it's already obvious from Github (it says this is a fork and gives the link to what this is a fork of already)